test/set-version-test.js

```diff
@@ -0,0 +1,83 @@
+"use strict";
+
+const grunt = require( "grunt" );
+const runTask = require( "grunt-run-task" );
+let tempFilePath = [ "tmp/testPackage.json" ];
+let expectedFilePath = [ "test/expected/testPackage.json" ];
+let noOfFiles = Math.min( tempFilePath.length, expectedFilePath.length );
```

Comment:
If this refers to 'number of files', please name this variable as such (i.e. `numberOfFiles`).